h5p-bildetema/language/nb.json

```diff
@@ -192,6 +192,126 @@
           "label": "Grid tekst",
           "default": "Enkeltbilder"
         },
+        {
+          "label": "Search",
```

Comment:
Labels can also be translated to the selected language. The labels don't have to be the same for each language, even though having them equal will off course make it easier to recognize each translation in the code. It's ok to keep English as the label language, but just want to note that most of the translations do have translated labels since this is most common in h5p. Could be something to discuss later, if we prefer to have English labels.

Reply:
Doesn't seem like the translated labels are visible anyway. So we can keep English

Reply:
I think English is best to maintain consistency and make it easier to add translations since all the labels will have the same key. Additionally, if the translator doesn't speak the language, it will be more challenging.